src/Framework/Framework/Resources/Scripts/utils/evaluator.ts

```diff
@@ -33,6 +34,44 @@ export function traverseContext(context: any, path: string): any {
     return currentLevel
 }
 
+export function findPathToChildObject(vm: any, child: any, path: string): string | null {
```

Comment:
Also... I quite don't like that this function operates on knockout observables instead of the state objects. The obserables are created lazily, because they are quite costly, so this will essentially populate the entire viewmodel